specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json

```diff
@@ -648,14 +651,31 @@
           "type": "string",
           "enum": [
             "Active",
-            "Restricted"
+            "Deleted",
```

Comment:
Why would someone see an IS in the deleted state? Wouldn't we simply not return the IS?